### DISCUSSION TITLE
`ghostty +list-actions` does not include those which do not have any doc comment attached

### DISCUSSION THREAD

**Discussion by 00-kat** [ghostty-org/ghostty#3879] [Feature Requests, Ideas] [answered]
Someone noted that these ones are grouped together with other ones on the website.  It appears to be even worse with the CLI: it doesn't even exist.

Exhaustive list: `text`, `paste_from_clipboard`, `paste_from_selection`, `decrease_font_size`, `scroll_to_bottom`, `scroll_page_up`, `scroll_page_down`, `scroll_page_fractional`, `scroll_page_lines`.

**Comment by mitchellh** [comment, marked as answer]
Issue #4958
